apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -144,12 +144,14 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
     if (!bookingReference || !bookingReference.bookingId) {
       log.error(
-        "bookingReference:",
+        "bookingReference not found error:",
         safeStringify({
           bookingReference,
+          room_name,
+          recording_id,
         })
       );
-      return res.status(404).send({ message: "Booking reference not found" });
```

Comment:
if daily webhook return non 20x(status code) response 3 consecutive times then it wouldn't get triggered again.

Reply:
we shouldn't block webhooks for everyone if it didn't work on some edge case.